misc/formatter/modules/formatter-core/src/test/resources/declarations/import/assert/import_declaration_2.bal

```diff
@@ -7,7 +7,7 @@ import ballerina/io as console;
 import ballerina/lang.'int;
 import ballerina/log as logger;
 import ballerina/math;
-// Ballerina extended library imports
+// Ballerina library imports
```

Comment:
Incorrect to have this import here as above lines (7,8,9) are also library imports.

Reply:
1. Corrected.